packages/workbox-recipes/src/warmStrategyCache.ts

```diff
@@ -0,0 +1,30 @@
+import { Strategy } from 'workbox-strategies/src/Strategy';
+
+import './_version.js';
+
+export interface WarmStrategyCacheOptions {
+  paths: Array<string>;
+  strategy: Strategy;
+}
+
+// Give TypeScript the correct global.
+declare let self: ServiceWorkerGlobalScope;
+
+/**
+ * 
```

Comment:
```suggestion
 * @memberof module:workbox-recipes
 
```

This should fix the documentation-related error. (Without it, JSDocs thinks that the function is in the global namespace.)